partition/gpt/table.go

```diff
@@ -322,6 +322,75 @@ func (t *Table) toGPTBytes(primary bool) ([]byte, error) {
 	return b, nil
 }
 
+// tableHeaderFromBytes read a partition table from a byte slice, mainly used to validate teh secondary header
+func tableHeaderFromBytes(b []byte, logicalBlockSize, physicalBlockSize int, skipMBR bool) (*Table, error) {
```

Comment:
This looks like it is almost completely duplicative of `tableFromBytes()`. Can we wrap these together to avoid code copies?

Reply:
Possibly.. there was some intertwinned data that would be hard to move around between the functions so I did a small dupe in the first iteration.

Reply:
Is it impossible to just have a single call?